### DISCUSSION TITLE
MVT export with multiple LAYER_CREATION_OPTIONS

### DISCUSSION THREAD

**Discussion by imakihi** [duckdb/duckdb-spatial#445] [Q&A]
Hi there. I tried to export my polygon layer as MVT format with multiple options, but I couldn't make it work. I wanted to specify minzoom, maxzoom, and compress options I could found here (https://gdal.org/en/latest/drivers/vector/mvt.html#driver-capabilities). 

I could pass one option when I generated MVT though.

This works.
```
COPY (select yoto_chiiki_name, st_transform(geom,'EPSG:4326','EPSG:3857',always_xy:=TRUE) geom from yoto) TO 'c:/tmp/yoto/yoto' WITH (FORMAT 'GDAL', DRIVER 'MVT', LAYER_CREATION_OPTIONS 'MAXZOOM=10');
```

This doesn't work
```
COPY (select yoto_chiiki_name, st_transform(geom,'EPSG:4326','EPSG:3857',always_xy:=TRUE) geom from yoto) TO 'c:/tmp/yoto/yoto' WITH (FORMAT 'GDAL', DRIVER 'MVT', LAYER_CREATION_OPTIONS 'MAXZOOM=10,MINZOOM=4');
```

I used a space, samicolon, and other separaters to define multiple options but nothing worked so far....

Can someone tell me how I should specify multiple options?

Thank you!

**Comment by fabito** [comment]
You need to pass a list like:

```sql
COPY (
  select 
    yoto_chiiki_name, 
    st_transform(geom,'EPSG:4326','EPSG:3857',always_xy:=TRUE) geom 
  from yoto
) TO 'c:/tmp/yoto/yoto' WITH (
  FORMAT 'GDAL', 
  DRIVER 'MVT', 
  LAYER_CREATION_OPTIONS ('MAXZOOM=10' , 'MINZOOM=4')
);
```
